pkgs/development/python-modules/celery-singleton/default.nix

```
@@ -54,15 +55,14 @@ buildPythonPackage rec {
 
   pytestFlagsArray = [ "tests" ];
 
-  # Tests require a running Redis backend
-  disabledTests = [
-    "TestLock"
-    "TestUnlock"
-    "TestClear"
-    "TestSimpleTask"
-    "TestRaiseOnDuplicateConfig"
-    "TestUniqueOn"
-  ];
+  preCheck = ''
+    ${redisServer}/bin/redis-server &
+    REDIS_PID=$!
+  '';
+
+  postCheck = ''
+    kill $REDIS_PID
+  '';
```

```suggestion
  '';

  __darwinAllowLocalNetworking = true;
```
We need this for the darwin test.
https://github.com/NixOS/nixpkgs/pull/259045/checks?check_run_id=17564080121